lib/facility_factory.rb

```diff
@@ -0,0 +1,47 @@
+class FacilityFactory
+
+    def initialize
+    end
```

Comment:
For classes that don't accept arguments upon instantiation, with an empty initialize, it's best to remove this method entirely. (`.new` will still work and assume no args will be passed)